packages/contractkit/src/wrappers/Governance.ts

```diff
@@ -381,13 +404,44 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
     })
   )
 
+  /**
+   * Returns the corresponding vote record
+   * @param voter Address of voter
+   * @param proposalID Governance proposal UUID
+   */
+  async getVoteRecord(voter: Address, proposalID: BigNumber.Value): Promise<VoteRecord | null> {
+    try {
+      const proposalIndex = await this.getDequeueIndex(proposalID)
+      const res = await this.contract.methods.getVoteRecord(voter, proposalIndex).call()
+      return {
+        proposalID: valueToBigNumber(res[0]),
+        value: Object.keys(VoteValue)[valueToInt(res[1])] as VoteValue,
+        votes: valueToBigNumber(res[2]),
+      }
+    } catch (_) {
+      // The proposal ID may not be present in the dequeued list, or the voter may not have a vote
+      // record for the proposal.
+      return null
+    }
+  }
+
   /**
    * Returns whether a given proposal is queued.
    * @param proposalID Governance proposal UUID
    */
   isQueued = proxyCall(this.contract.methods.isQueued, tupleParser(valueToString))
 
-  /**
+  /*
```

Comment:
Looks like ** is needed, otherwise it doesn't go to  docs.

Reply:
Oops!